composer.json

```diff
@@ -16,9 +16,9 @@
         }
     ],
     "require": {
-        "php": "<7.2.0",
+        "php": ">=5.6",
```

Comment:
Maybe just remove this - core defines the default minimum anyway

Reply:
OK gone :)